README.md

```diff
@@ -118,6 +118,11 @@ are available
 * `CLAP_MANUAL_URL` and `CLAP_SUPPORT_URL` generate the urls in your description
 * `CLAP_MISBHEAVIOUR_HANDLER_LEVEL` can be set to `Terminate` or `Ignore` (default
   is `Ignore`) to choose your behaviour for a misbehaving host.
+* `CLAP_CHECKING_LEVEL` can be set to `None`, `Minimal`, or `Maximal` (default is
+  `Minimal`) to choose the error checking level for the plugin.
```

Comment:
Is this documentation correct? I had a tough time finding comments about what exactly `CheckingLevel` is supposed to do.

Reply:
Yeah you sort of have to read the code it’s not that well documented 

basicalky the plugin helpers toggle on more and more sanity checks as these levels go up but what they are is sort of up to Alex so far